app/models/user.rb

```diff
@@ -36,7 +36,7 @@ class User < ApplicationRecord
 
   validates_presence_of   :name, :userid
   validates :userid, :uniqueness => {:conditions => -> { in_my_region } }
-  validates_format_of     :email, :with => /\A([\w\.\-\+]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i,
+  validates_format_of     :email, :with => /\A([\w\.\-\+']+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i,
```

Comment:
@imtayadeway @gtanzillo Maybe [this regex](https://github.com/ManageIQ/more_core_extensions/blob/master/lib/more_core_extensions/core_ext/string/formats.rb#L4) should be used, instead?